tests/test_kubernetes_tools.py

```diff
@@ -2773,6 +2781,25 @@ def test_get_persistent_volume_name(self):
         )
         assert pv_name == "pv--slash-blahslash-what"
 
+    @mock.patch(
+        "paasta_tools.kubernetes_tools.load_system_paasta_config", autospec=None
+    )
+    @mock.patch(
+        "paasta_tools.kubernetes_tools.get_authenticating_services", autospec=None
+    )
+    def test_get_projected_sa_volumes_token_automount(
+        self, mock_get_auth_services, mock_system_config
+    ):
+        mock_get_auth_services.return_value = {"service_a", "service_b", "kurupt"}
```

Comment:
it's a little hard to follow why the test has these three services that are authenticating but the assertion is for only a single projected sa volume. Can you clarify the test setup / expectations with comments?